### PR TITLE
[sysdig] add customizable envvars for NodeAnalyzer deployments

### PR DESCRIPTION
**Pull request by r2bit**
## What this PR does / why we need it:
Recently, while working on a performance issue with Sysdig support, we
discovered that tuning RuntimeScanner's memory settings fixed the issue.
However, that tunable is set via environment variables and current Chart
does not allow setting custom envvars.
Instead of having to wait until that specific tunable is added to the
Chart, I think it's better to add a broad support for custom envvars for
just such occasions.

This PR adds `env` object to:
* imageAnalyzer
* hostAnalyzer
* benchmarkRunner
* runtimeScanner
* cspmAnalyzer
* cspmCollector

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

### CODE REVIEW

**Review by yashgiri — COMMENTED** [review]
LGTM

File: charts/sysdig/values.yaml [hunk 1]
```diff
@@ -553,6 +565,8 @@ cspmCollector:
       cpu: 500m
       memory: 1536Mi
 
+  env: { }
```
Comment (lilx1ao) [inline charts/sysdig/values.yaml]: we do not leave spaces in between the curly brackets as the default value placeholder. check out the example below from line 556/570.

**Comment by achandras** [comment]
@r2bit thanks for this PR! Could you please bump the version in `Chart.yaml`?

**Comment by r2bit** [comment]
> @r2bit thanks for this PR! Could you please bump the version in `Chart.yaml`?

Hope patch bump is enough. :)

**Comment by achandras** [comment]
Thanks!

**Review by achandras — APPROVED** [review]

**Pull request merged**